innotter/views.py

```diff
@@ -0,0 +1,68 @@
+import os
+from datetime import datetime, timedelta
+
+import jwt
+from django.contrib.auth import authenticate
+from rest_framework import mixins, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticatedOrReadOnly, AllowAny, IsAuthenticated, IsAdminUser
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from innotter.models import Page, User
+from innotter.serializers import PageSerializer, UserSerializer, AdminOrModeratorPageSerializer, OwnerPageSerializer
+from innotter.services.permissions.permissions import IsOwnerOrModeratorCanEdit
+
+
+class PageViewSet(mixins.ListModelMixin,
+                  mixins.CreateModelMixin,
+                  mixins.RetrieveModelMixin,
+                  mixins.UpdateModelMixin,
+                  mixins.DestroyModelMixin,
+                  GenericViewSet):
+
+    queryset = Page.objects.all()
+    serializer_class = PageSerializer
+    permission_classes = [IsOwnerOrModeratorCanEdit, IsAuthenticated]
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+
+    @action(detail=False, methods=['post'], permission_classes=[AllowAny])
+    def register(self, request):
+        serializer = UserSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=201)
+        else:
+            return Response(serializer.errors, status=400)
```

Comment:
```suggestion
        if serializer.is_valid(raise_exception=True):
            serializer.save()
            return Response(serializer.data, status=201)
```
You can use raise_exception=True instead and it will do the same